_topic_map.yml

```diff
@@ -1275,23 +1275,23 @@ Name: Monitoring
 Dir: monitoring
 Distros: openshift-enterprise,openshift-webscale,openshift-origin
 Topics:
-- Name: Cluster monitoring
-  Dir: cluster_monitoring
-  Topics:
-  - Name: About cluster monitoring
-    File: about-cluster-monitoring
-  - Name: Configuring the monitoring stack
-    File: configuring-the-monitoring-stack
-  - Name: Managing cluster alerts
-    File: managing-cluster-alerts
-  - Name: Examining cluster metrics
-    File: examining-cluster-metrics
-  - Name: Accessing Prometheus, Alertmanager, and Grafana
-    File: prometheus-alertmanager-and-grafana
-- Name: Monitoring your own services
-  File: monitoring-your-own-services
+- Name: Understanding monitoring
+  File: understanding-monitoring
+- Name: Configuring platform monitoring
```

Comment:
Why calling it "Platform Monitoring" when you really configure the OpenShift Monitoring stack? I know we still think in terms of "Platform" vs "User Workload" but we should really start to think about this as a single Monitoring stack that now has two main goals: 1) providing a turn-key experience to support K8s services and best practices Monitoring K8s + OpenShift components and 2) provide a way for anyone to monitor their own workload/infrastructure with a multi-tenant experience.    

Shouldn't that section really cover how to configure the Monitoring stack? I know that there are still two Prometheus cluster but most options are exactly the same and we could explain where the differences are by laying out the scenarios. WDYT?

Reply:
Based on your comment I think we should rename this 'Configuring the monitoring stack'. Does that sound suitable to you?

Reply:
Yes.

Reply:
I have now renamed this title as discussed.